src/api/models/customer.py

```diff
@@ -0,0 +1,149 @@
+# coding: utf-8
+"""Customer Model."""
+
+from __future__ import absolute_import
+
+# Standard Python Libraries
+from datetime import date, datetime  # noqa: F401
+from typing import Dict, List  # noqa: F401
+
+# cisagov Libraries
+from api import util
+from api.models.base_model_ import Model
+
+
+class Customer(Model):
+    """NOTE: This class is auto generated by the swagger code generator program.
+
+    Do not edit the class manually.
+    """
+
+    def __init__(
+        self,
+        uuid: str = None,
+        name: str = None,
+        contact: str = None,
+        status: str = None,
+    ):
+        """Initialize Customer model defined in Swagger.
+
+        :param uuid: The uuid of this Customer.  # noqa: E501
+        :type uuid: str
+        :param name: The name of this Customer.  # noqa: E501
+        :type name: str
+        :param contact: The contact of this Customer.  # noqa: E501
+        :type contact: str
+        :param status: The status of this Customer.  # noqa: E501
+        :type status: str
+        """
+        # noqa: E501
+        self.swagger_types = {
+            "uuid": str,
+            "name": str,
+            "contact": str,
+            "status": str,
+        }  # type: ignore
```

Comment:
More unnecessary `E501` and `type: ignore`s that should be taken care of here and below.

Reply:
addressed in b0be2dc

Reply:
The `E501`s are taken care of here, but I'm going to leave this conversation open until the `type: ignore`s have also been sorted out.

Reply:
Sounds good. I'm working on the cleanup now and will reference the commit here once it's pushed up. There shouldn't be a need for any linter exceptions, honestly. We should address the root issues now to keep things clean.